flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricNames.java

```diff
@@ -67,6 +67,7 @@ private MetricNames() {
 
 	public static final String CHECKPOINT_ALIGNMENT_TIME = "checkpointAlignmentTime";
 	public static final String CHECKPOINT_START_DELAY_TIME = "checkpointStartDelayNanos";
+	public static final String WORKER_FAILURE_RATE = "workFailureRate";
```

Comment:
```suggestion
	public static final String WORKER_FAILURE_RATE = "startWorkFailure" + SUFFIX_RATE;
```

Reply:
Not sure if per-second is a good granularity for this metric.
Maybe it's not a big deal since people can always convert it to other granularity.

Reply:
Done.